contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -20,7 +23,6 @@
         <Banner :value="!valid" error class="mb-4">
```

Comment:
Could this alert be moved on top, right after the main heading `Create and account`, same as it is on the sign in page, right after the Kolibri Studio heading?

Reply:
Maybe they need to restart the browser or the device to get the connection back, and there's nothing more annoying than having to fill in the same (long) form again... 😒

Reply:
to echo some of Radina's thoughts above, I'd also prefer the alert moved after the 'create an account' heading and remove the offline icon indicator. The registration form on Studio is fairly long and there's a change the user might miss the offline indicator regardless if they're viewing the top or bottom portion of the screen. 

Ideally I'd like to repurpose and tweak this behavior in Kolibri for these two cases:
- Repurpose the sticky behavior of the snackbar below, so wherever the user is on the form, the error will be visible
- Remove the dark scrim background so the user can still input information into the text fields but wait until their connection is restored to submit

<img width="592" alt="Screen Shot 2021-05-28 at 6 07 56 PM" src="https://user-images.githubusercontent.com/25045166/120053649-38b68f80-bfe0-11eb-9d35-2d2f4092a619.png">

<img width="567" alt="Screen Shot 2021-05-28 at 6 08 50 PM" src="https://user-images.githubusercontent.com/25045166/120053651-39e7bc80-bfe0-11eb-80ce-f1aacbd83d99.png">

Let me know if this is possible, @micahscopes and I'll mock something up for it!

Reply:
I agree with the proposed sticky snackbar, good strategy to make the alert visible independent of where the user is along the form. 

Also, as I suggested, let's rephrase the string to `You seem to be offline. Please connect to the internet to create an account.` to avoid introducing the the term `registration`.

Reply:
Thanks for the feedback @radinamatic and @khangmach.  

When I first got into this, a snackbar seemed ideal, but using the actual snackbar components appeared to be tricky when I tried before, just because these login pages aren't part of the same Vue app as the rest of Studio.  

That said, I'm glad to take another look at that.  In the meantime, could simply moving the message up to the top be a reasonable starting point to get the basic functionality in for now, with the intention of iterating and switching to snackbars going forward?

Reply:
@micahscopes yep we can do that in the meantime.

Reply:
👍🏽 from me too, just correct the strings so we can have one thing less to think about!